src/components/datetime/DateTimeController.cpp

```diff
@@ -120,6 +121,10 @@ const char* DateTime::MonthShortToStringLow(Months month) {
   return MonthsStringLow[static_cast<uint8_t>(month)];
 }
 
+const char* DateTime::DayOfWeekShortToStringLow() {
```

Comment:
just as in https://github.com/InfiniTimeOrg/InfiniTime/pull/1039 this function can be const

```suggestion
const char* DateTime::DayOfWeekShortToStringLow() const {
```